### PR TITLE
added various modifications

### PR DESCRIPTION
**Pull request by cissoucr**

### CODE REVIEW

File: docs/tools/ColabFold.md [hunk 1]
```diff
@@ -70,17 +70,19 @@ The code below is an example of a batch script to run MMSeqs2. The top of the sc
 
 module load uri
 module load MMseqs2/14-7e284-gompi-2021b
+module load cudnn/cuda11-8.4.1.50
+module load cuda/11.4.0
 module load miniconda
```
Comment (georgiastuart) [inline docs/tools/ColabFold.md:75]: Add the miniconda version

File: docs/tools/ColabFold.md [hunk 2]
```diff
@@ -70,17 +70,19 @@ The code below is an example of a batch script to run MMSeqs2. The top of the sc
 
 module load uri
```
Comment (georgiastuart) [inline docs/tools/ColabFold.md:71]: does the `uri` module also require a version now?

File: docs/tools/ColabFold.md [hunk 3]
```diff
@@ -103,10 +105,10 @@ The parameter `--stop-at-score` is used to stop generating models until the pred
 
 ```bash
 #!/bin/bash
-#SBATCH --partition=uri-gpu
+#SBATCH --partition=gpu,gpu-preempt,gpu-long
 #SBATCH --nodes=1
-#SBATCH --cpus-per-gpu=16
-#SBATCH -mem-per-gpu=80
+#SBATCH --gpus-per-node=1
+#SBATCH --mem-per-gpu=11
```
Comment (georgiastuart) [inline docs/tools/ColabFold.md:111]: Is this accurate? Mem is referring to cpu memory, not vram as it seems above.

File: docs/tools/ColabFold.md [hunk 4]
```diff
@@ -140,6 +142,8 @@ The next 2 files are generated for the 5 trained models:
 #### Notes:
 * `<path to directory containing MSAs>` is the same as `<path to the output directory>` used with the `colabfold_search` command.
 * `<path to output directory>` is the full path to an existing directory used to store the results.
+* When dealing with a large number of sequences, it is recommended to sort proteins into batches based on their size and submit a job to a GPU node with smaller VRAM for batches with shorter proteins.
```
Comment (georgiastuart) [inline docs/tools/ColabFold.md:145]:
```suggestion
* When dealing with a large number of sequences, we recommended sorting proteins into batches based on their size and submitting a job to a GPU node with smaller VRAM for batches with shorter proteins.
```

**Review by georgiastuart — CHANGES_REQUESTED** [review]
Just a few small things. Looks great!

File: docs/tools/ColabFold.md (hunk 1 above)
Comment (cissoucr) [reply docs/tools/ColabFold.md:75]: Good catch!

File: docs/tools/ColabFold.md (hunk 2 above)
Comment (bryank-cs) [reply docs/tools/ColabFold.md:71]: @cissoucr This should be uri/main now.

**Review by georgiastuart — APPROVED** [review]
Looks great!

**Pull request merged**